zebrad/tests/common/lightwalletd/proto/service.proto

```diff
@@ -36,7 +36,7 @@ message TxFilter {
 // by GetMempoolStream(), the latest block height.
 message RawTransaction {
```

Comment:
Updating this file seems like something that's easy to forget. Is there somewhere we can document that it needs to be done?

Reply:
Added a comment in https://github.com/ZcashFoundation/zebra/pull/7644/commits/b4e6761673620a5720f0bdcbe9b2ae27944cd43b.